tests/components/device_automation/test_init.py

```diff
@@ -546,14 +580,16 @@ def _same_lists(a, b):
                     "device_id": "none",
                     "domain": "test",
                 },
-                "action": {"service": "test.automation", "entity_id": "hello.world"},
+                "action": {
+                    "service": "test.automation",
+                    "entity_id": "hello.world"
+                },
             }
         },
     )
 
-    assert (
-        "Integration 'test' does not support device automation triggers" in caplog.text
-    )
+    assert ("Integration 'test' does not support device automation triggers" in
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)